core/performance_test/peakflops/CMakeLists.txt

```diff
@@ -1,48 +1,44 @@
 # It only makes sense to run this if testing is enabled, as we depend on gtest
 if(NOT Cabana_ENABLE_TESTING)
+  message(WARNING "Cabana_ENABLE_TESTING disabled, skipping performance tests")
   return()
 endif()
 
-set(Cabana_PERFORMANCE_EXPECTED_FLOPS "" CACHE STRING "Value for expected flops in performance tests")
-set(Cabana_PERFORMANCE_ERROR_MARGIN "" CACHE STRING "Error margin to use in performance tests")
-set(Cabana_PERFORMANCE_VECLENGTH "" CACHE STRING "Vector Length to use when running expected flops")
-set(Cabana_PERFORMANCE_ITERATIONS "" CACHE STRING "Number of iterations to repeat the test for (increases flops)")
-set(Cabana_PERFORMANCE_SEED "" CACHE STRING "Seed for performance test RNG")
+set(Cabana_PERFORMANCE_EXPECTED_FLOPS "0" CACHE STRING "Value for expected flops in performance tests")
+set(Cabana_PERFORMANCE_ERROR_MARGIN "0" CACHE STRING "Error margin to use in performance tests")
```

Comment:
I think the defaults should probably match what's in the code, here:

https://github.com/ECP-copa/Cabana/blob/dd7101b63411cad5208dad0299e7c417ea2a7dcf/core/performance_test/peakflops/common.h

 and 

https://github.com/ECP-copa/Cabana/blob/0f4de9cad7feadc52d1ee15e11fd0b51f7294e78/core/performance_test/peakflops/01_cpp_simple.cpp#L140

If we're hoisting the default to here, we should probably tidy those bits of the code too?

Reply:
Much cleaner, but it does now use the same number of iterations per test